src/utils/context_helper.ts

```diff
@@ -45,6 +44,14 @@ export async function buildContexts(params: any) {
     contexts.push(_pluginPayload);
   }
 
+  // uncommitted_changes
+  if (params.uncommitted_changes?.length > 0) {
```

Comment:
If you just need to check that it's an array, you can do
```
Array.isArray(params.uncommitted_changes)
```

Reply:
I just need to check for existence really.   the length part is unnecessary, since the iterator will just do nothing if the array is empty.

I suppose I could just do ```if (params.uncommitted_changes) {```

Reply:
Actually the Array check works for undefined properties too:

```
> params = {}
{}

> Array.isArray(params.uncommitted_changes)
false
```